src/dbt_osmosis/main.py

```diff
@@ -241,6 +259,9 @@ def organize(
     check: bool = False,
     models: Optional[List[str]] = None,
     skip_add_columns: bool = False,
+    skip_add_tags: bool = False,
```

Comment:
This line is not directly related to this pull request, but I forgot to add it when I was working on https://github.com/z3z1ma/dbt-osmosis/pull/78, so I'm fixing it here together 🙏 .